docs/examples/dns/auroradns/enable_disable_record.py

```diff
@@ -0,0 +1,13 @@
+from libcloud.dns.types import Provider
+from libcloud.dns.providers import get_driver
+
+cls = get_driver(Provider.AURORADNS)
+
+driver = cls('myapikey', 'mysecret')
+
+zone = driver.get_zone("auroradns.eu")
+
+record = zone.create_record(name="www", type="AAAA", data="2a00:f10:452::1",
+                            extra={"disabled": True})
```

Comment:
A better alternative than using `extra` dict would be to use `ex_disabled=False` (or `ex_enabled=True`) extension argument.